lib/controller.js

```diff
@@ -946,11 +946,22 @@ controller.tesselFirmwareVerion = function(opts) {
         return updates.requestBuildList().then(function(builds) {
           // Figure out what commit SHA is running on it
           var version = updates.findBuild(builds, 'sha', versionSha).version;
-          logs.info('Tessel [' + tessel.name + '] version: ' + version);
+          var tesselVersion = 'Tessel [' + tessel.name + '] version: ' + version;
+
+          return tessel.fetchCurrentNodeVersion()
+            .then(nodeVersion => {
+              logs.info(tesselVersion);
+              logs.info('Tessel [' + tessel.name + '] Node version: ' + nodeVersion);
+            })
+            .catch(() => {
+              logs.info(tesselVersion);
+              logs.info('Tessel [' + tessel.name + '] Node version: unknown');
+            });
         });
       })
-      .catch(function() {
+      .catch(() => {
         logs.info('Tessel [' + tessel.name + '] version: unknown');
+        logs.info('Tessel [' + tessel.name + '] Node version: unknown');
       });
   });
 };
```

Comment:
The more I think about it, the more I wish this was a serializable format, or at very least super easy to parse